Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 
+// swiftlint:disable type_body_length file_length
```

Comment:
Random - do we want to revisit some of our lint rules? Or start refactoring to abide by the rules sometime? I notice these coming in more often

Reply:
We _should_ be able to remove both of these once we remove the old init. I think the new init pushed us over the limit, but agree we should have a plan for addressing linter rules we seem to disable often.